setup.py

```diff
@@ -106,7 +106,7 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['numpy', 'astropy', 'astropy-healpix', 'scipy'],
+      install_requires=['numpy', 'astropy >= 1.0', 'astropy-healpix', 'scipy'],
```

Comment:
I think the spaces around the >= should be removed

Reply:
Done